docs/community.vmware.vmware_guest_disk_module.rst

```diff
@@ -83,7 +83,13 @@ Parameters
                         <div>- <code>thin</code> thin disk</div>
                         <div>- <code>eagerzeroedthick</code> eagerzeroedthick disk</div>
                         <div>- <code>thick</code> thick disk</div>
+                        <div>- <code>rdm</code> raw device mapping</div>
                         <div>Default: <code>thick</code> thick disk, no eagerzero.</div>
+                        <div>- <code>compatibility_mode</code> (string): Valid values are:</div>
```

Comment:
This is autogenerated by https://github.com/ansible-network/collection_prep/blob/440fc5138f7b12a17df0689bafaa073ed14eb6e7/collection_prep/cmd/add_docs.py#L444

You may want to run command like - 

```
$ git clone https://github.com/ansible-network/collection_prep/
$ cd ./collection_prep/collection_prep
$ python cmd/add_docs.py -p <PATH_TO_COLLECTION>/collections/ansible_collections/community/vmware -b main
```

Reply:
I went ahead and undid that change. I ran the collection_prep but it changed more of the rst files than I was comfortable with. Let me know if there is a way to limit it to just the module.